crypto/s2n_ecc_evp.c

```diff
@@ -480,21 +490,24 @@ int s2n_ecc_evp_parse_params_point(struct s2n_blob *point_blob, struct s2n_ecc_e
     return 0;
 }
 
-int s2n_ecc_evp_parse_params(struct s2n_connection* conn,
-                             struct s2n_ecdhe_raw_server_params* raw_server_ecc_params,
-                             struct s2n_ecc_evp_params* ecc_evp_params) {
-    POSIX_ENSURE(
-            s2n_ecc_evp_find_supported_curve(conn, &raw_server_ecc_params->curve_blob, &ecc_evp_params->negotiated_curve) == 0,
+int s2n_ecc_evp_parse_params(struct s2n_connection *conn, struct s2n_ecdhe_raw_server_params *raw_server_ecc_params,
+        struct s2n_ecc_evp_params *ecc_evp_params)
+{
+    POSIX_ENSURE(s2n_ecc_evp_find_supported_curve(conn, &raw_server_ecc_params->curve_blob,
+                         &ecc_evp_params->negotiated_curve)
+                    == 0,
```

Comment:
Yeah, this looks weird. Is this fixable or is the price we pay for turning on a formatter?

Reply:
This is totally flexible. We can choose what goes on what line. ColLimit: 0 effectively turns off the the automatic line wrapping. We can put it all on one line or many.